homeassistant/components/qbus/coordinator.py

```diff
@@ -0,0 +1,274 @@
+"""Qbus coordinator."""
+
+from __future__ import annotations
+
+from datetime import datetime
+import logging
+from typing import cast
+
+from qbusmqttapi.discovery import QbusDiscovery, QbusMqttDevice, QbusMqttOutput
+from qbusmqttapi.factory import QbusMqttMessageFactory, QbusMqttTopicFactory
+
+from homeassistant.components.mqtt import (
+    ReceiveMessage,
+    async_wait_for_mqtt_client,
+    client as mqtt,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import EVENT_HOMEASSISTANT_STOP
+from homeassistant.core import CALLBACK_TYPE, Event, HomeAssistant
+from homeassistant.helpers import device_registry as dr
+from homeassistant.helpers.device_registry import format_mac
+from homeassistant.helpers.event import async_call_later
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator
+from homeassistant.util.hass_dict import HassKey
+
+from .const import CONF_SERIAL_NUMBER, DOMAIN, MANUFACTURER
+
+_LOGGER = logging.getLogger(__name__)
+
+
+type QbusConfigEntry = ConfigEntry[QbusControllerCoordinator]
+QBUS_KEY: HassKey[QbusConfigCoordinator] = HassKey(DOMAIN)
+
+
+class QbusControllerCoordinator(DataUpdateCoordinator[list[QbusMqttOutput]]):
+    """Qbus data coordinator."""
+
+    _WAIT_TIME = 3
```

Comment:
```suggestion
    _STATE_REQUEST_DELAY = 3
```
I think this makes it clearer what it is used for